static/src/stylesheets/inline/article-labour-liverpool.scss

```diff
@@ -335,3 +395,32 @@ $labour-liverpool-header-element-opacity: .85;
     z-index: 40;
     color: $news-main-1;
 }
+
+.heroic-video {
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 60px;
+
+    @include mq($from: tablet) {
+        width: 740px;
+        margin-top: 0px;
+    }
+
+    @include mq($from: desktop) {
+        width: 980px;
+    }
+}
+
+.l-side-margins {
+    &.heroic--video {
+        &::before {
+            margin-top: -560px;
+            height: 128%;
```

Comment:
Can you add a comment here just so we know what's going on when we come back to it?